compiler/noirc_frontend/src/hir/comptime/errors.rs

```diff
@@ -683,6 +687,13 @@ impl<'a> From<&'a InterpreterError> for CustomDiagnostic {
                 let secondary = String::new();
                 CustomDiagnostic::simple_error(msg, secondary, location.span)
             }
+            InterpreterError::LoopHaltedForUiResponsiveness { location } => {
+                let msg = "This loop took too much time to execute so it was halted for UI responsiveness"
+                    .to_string();
+                let secondary =
+                    "This error doesn't happen in normal executions of `nargo`".to_string();
+                CustomDiagnostic::simple_error(msg, secondary, location.span)
```

Comment:
This should be a warning I think since its not something actually wrong with the program